### ISSUE TITLE
Support for mediarithmics analytics

### ISSUE THREAD

**Issue by PoulpiFr** [ampproject/amphtml#11262]
## What's the feature?

mediarithmics ([see here](http://www.mediarithmics.com/en/)) is providing to its customers a Data Marketing platform.

This platform includes many different parts, including a analytics and DMP modules.

mediarithmics recently became a provider for many French newspapers that are already using AMP or are thinking to switch to it.

We would like to add our entry to the vendor.js part of amp-analytics so that the integration with AMP would become easier for our customers.

**Comment by jridgewell** [comment]
Awesome! to @avimehta

**Comment by PoulpiFr** [comment]
Hello,

Sorry for the delay, I didn't have time in the last days to make progress on this.

I just opened a PR with an integration proposal. It is blocked because:
- We just signed the Google CLA and Github is not aware yet. Is there something to do to force the refresh? is it automatic?
- Apparently, some Percy tests are failing but I don't have the right to access Percy => Is there blocking issues there?
